File: src/main/java/com/dh/ondot/schedule/domain/repository/AiUsageRepository.java
package com.dh.ondot.schedule.domain.repository;

import com.dh.ondot.schedule.domain.AiUsage;
import org.springframework.data.jpa.repository.JpaRepository;

import java.time.LocalDate;
import java.util.Optional;

public interface AiUsageRepository extends JpaRepository<AiUsage, Long> {
    Optional<AiUsage> findByMemberIdAndUsageDate(Long memberId, LocalDate usageDate);
}

File: src/main/kotlin/com/dh/ondot/schedule/domain/repository/AiUsageRepository.kt
package com.dh.ondot.schedule.domain.repository

import com.dh.ondot.schedule.domain.AiUsage
import org.springframework.data.jpa.repository.JpaRepository
import java.time.LocalDate
import java.util.Optional

interface AiUsageRepository : JpaRepository<AiUsage, Long> {
    fun findByMemberIdAndUsageDate(memberId: Long, usageDate: LocalDate): Optional<AiUsage>
}
